src/classes/RD2_OpportunityMatcher.cls

```diff
@@ -216,10 +216,12 @@ public inherited sharing class RD2_OpportunityMatcher {
             matched.attachInstallment(installment);
             return;
         }
-        
+
         //if the Installment period is not monthly, do not attempt to match by +/- match days range
         //and add the installment without matching Opp
-        if (installment.installmentPeriod != RD2_Constants.INSTALLMENT_PERIOD_MONTHLY) {
+        if (installment.installmentPeriod != RD2_Constants.INSTALLMENT_PERIOD_MONTHLY &&
+            installment.installmentPeriod != RD2_Constants.INSTALLMENT_PERIOD_YEARLY) {
```

Comment:
Can extract this as a method on the Installment class.

Reply:
@bdvorachek, thinking through this one a little more, I think moving this to the Installment class might be a little tricky. The reason is that this boolean's result is determining matching behavior that belongs to this Matcher class. I was thinking this boolean condition could determine a certain type of Installment so that the Installment class could know what type it is. However, that would only work if that type is useful or valid outside of needing it for matching behavior. That being said, I'm not sure that it would belong in the Installment class unless there's a way it could be decoupled from matching behavior

Reply:
Would this be better as a method on this class that accepts an installment as an argument then? I think there's still value in giving this conditional a name.

Reply:
Just pushed up a change to extract that conditional logic into a method